roles/openshift_logging_curator/tasks/main.yaml

```diff
@@ -92,12 +102,12 @@
     curator_name: "{{ 'logging-curator' ~ ( (openshift_logging_curator_ops_deployment | default(false) | bool) | ternary('-ops', '') ) }}"
     curator_component: "{{ 'curator' ~ ( (openshift_logging_curator_ops_deployment | default(false) | bool) | ternary('-ops', '') ) }}"
 
-# DC
```

Comment:
We should remove the old DC as part of setting up the cron job

Reply:
Per a discussion in scrum -- there is no need to do this. there is little to no harm with the running at the same time. In fact, curator for 2.x may not even work for es 5.x